### DISCUSSION TITLE
modificare entità

### DISCUSSION THREAD

**Discussion by fabiofi75** [indomus/forum#1118] [GENERALE - Discussioni generiche]
Ho un problema con le entità, ho installato sonoff e shelly per comandare tutte le luci di casa, le ho integrate in home assistant ma le riconosce tutte come switch. Come faccio a definirle come light e differenziarle dai reali switch?

**Comment by xyzmauro** [comment]
Ciao,dai un occhio [qui](https://www.home-assistant.io/integrations/light.switch/)
